frontend/src/components/Board/Board.tsx

```diff
@@ -1,7 +1,327 @@
-import React from 'react';
+import { useState } from "react";
+// import { AccountCircle } from "@mui/icons-material";
+import { DragDropContext, Draggable, Droppable } from "@hello-pangea/dnd";
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from "@mui/material";
 
-export const Board: React.FC = () => (
-  <>
-    <h1>This is the board component</h1>
-  </>
-);
+// Mock data
+const board = {
+  id: "123456789",
+  version: 1,
+  saved_date: new Date().toDateString(),
+  created_at: new Date().toDateString(),
+  user_id: "1111",
+  team_id: "2222",
+  name: "FIT3162 Board",
+  description:
+    "This board is used to monitor the progress of the FIT3162 CS_20 Project.",
+};
+
+// list table
+const columns: IColumn[] = [
```

Comment:
lets put the types in a seperate d.ts file. cleans up & we dont need to import types too if we use d.ts